### PR TITLE
Added 'getCountries' method to 'CLocale' class

### PR DESCRIPTION
**Pull request by ua2004**
Method 'getCountries' returns array of all country names in current system language.

### CODE REVIEW

**Comment by samdark** [comment]
Please format code according to 1.1 style and add a line for changelog.

**Comment by ua2004** [comment]
@samdark could you please specify what exactly should be formatted? I tried to keep the original formatting.

File: framework/i18n/CLocale.php [hunk 1]
```diff
@@ -469,4 +469,22 @@ public function getTerritory($id)
...
 		return $this->getLocaleDisplayName($id, 'territories');
 	}
+	
+	/** 
+	 * @return array list of local display names of all countries.
+	 * @since 1.1.18
+	 */
+	public function getCountries()
+	{
+		$countries = array();
```
Comment (samdark) [inline framework/i18n/CLocale.php]: There should be no spaces around `=`.

File: framework/i18n/CLocale.php [hunk 2]
```diff
@@ -469,4 +469,22 @@ public function getTerritory($id)
...
+		foreach($this->_data['territories'] as $code => $country)
+		{
+			//zz is an unknown region
+			if(($code != 'zz') && preg_match('/[a-z]{2}/', $code))
```
Comment (samdark) [inline framework/i18n/CLocale.php]: There should be no spaces around `!=` and after `,`.

File: framework/i18n/CLocale.php [hunk 3]
```diff
@@ -469,4 +469,22 @@ public function getTerritory($id)
...
+			{
+				$countries[$code] = $country;
```
Comment (samdark) [inline framework/i18n/CLocale.php]: There should be no spaces around `=`.

**Comment by samdark** [comment]
Closed since no enhancements are accepted in 1.1 anymore.

**Pull request closed without merging**